examples/advanced/flutter/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [0.0.1] - TODO: Add release date.
+
+* TODO: Describe initial release.
```

Comment:
Is this a placeholder file that someone who generates flutter would need to fill out before distribution of the generated assets? If so, it might be good to give them a bit of description to tell them the purpose of the file / be explicit in what to do. Maybe this is well known to flutter folk, but to me I was unsure if these were for us TODO.

If this is not a placeholder file for users to fill out, lets delete it.

Reply:
So, the CHANGELOG.md file is used automatically by flutter's package manager (pub.dev) when you publish a package there.
I will delete it, since this package will not be published to be used as a dependency anyway.